internal/backend/svc/svc.go

```diff
@@ -140,6 +140,20 @@ func makeFxOpts(cfg *Config, opts RunOptions) []fx.Option {
 				HookOnStart(lc, c.Start)
 				HookOnStop(lc, c.Stop)
 			}),
+			fx.Invoke(func(cfg *temporalclient.Config, tclient temporalclient.Client, muxes *muxes.Muxes) {
+				if !cfg.EnableHelperRedirect {
+					return
+				}
+
+				_, uiAddr := tclient.TemporalAddr()
```

Comment:
just to ensure - what are the cases when uiAddr is empty? when temporal started by us, but without ui?

Reply:
yup, banner knows how to deal with it